### DISCUSSION TITLE
Applicability

### DISCUSSION THREAD

**Discussion by KALIPEM** [smol-ai/GodMode#45] [Q&A] [answered]
I wanted to ask whether I will be able to use this to further develop existing apps, extensions, webpages, and web apps, which are functional and running. like feeding the ai the existing codes and giving a promo to add another feature or remove one or replace one or just simply add more content to it, or is it just for building basic structure of the project based on your idea of the project?

**Comment by swyxio** [comment, marked as answer]
hmm just for basic structure!
